### ISSUE TITLE
Full filtering on icephys hierarchical tables

### ISSUE THREAD

**Issue by lvsltz** [NeurodataWithoutBorders/nwb-jupyter-widgets#55]
**Introduction**
This is just an idea on how to visualize the icephys data when hierarchical tables [from ndx-icephys-meta](https://github.com/oruebel/ndx-icephys-meta) are present.
This will:
- probably cover all queries listed in [this notebook's Section 3](https://github.com/oruebel/ndx-icephys-meta/blob/master/src/pynwb/examples/icephys_meta_query_example.ipynb).
- replace all tables present now:
![image](https://user-images.githubusercontent.com/6717189/74344891-8b4d4780-4dad-11ea-91f6-a79430e16bd4.png)
- make obsolete issue https://github.com/NeurodataWithoutBorders/nwb-jupyter-widgets/issues/54  

**Proposal**
Create listboxes/lists/buttons that allow the user to select one or more items from the lists of unique
`cells`, `conditions`, `runs`, `sweep_sequences`, `sweeps`, `intracellular_recordings`, `[stimulus, response]` and dynamically show the plots corresponding to the intersection of selected features.

**Example**
User selects `condition = 1`,  `sweep_sequence = A` and `response`.
The responses of all sweeps from all cells and all repetitions will be plotted.

**Comment by bendichter** [comment]
I think I understand. Do you also want stimulus_type as an option?

**Comment by lvsltz** [comment]
Yes. I didn't explain it properly, but that's what I actually meant by adding `sweep_sequences`.
And `repetition` from the `runs` table.

**Comment by bendichter** [comment]
got it

**Comment by bendichter** [comment]
My plan is to combine this with #54, giving the user the full tree to navigate through, but also giving them dropdown menus that will automatically forward branches of the tree when selected.
